src/main/kotlin/nl/avisi/kotlinwebtest/jdbc/Jdbc.kt

```diff
@@ -0,0 +1,149 @@
+/**
+ * For licensing, see LICENSE.txt
+ * @author Martijn Heuvelink
+ */
+package nl.avisi.kotlinwebtest.jdbc
+
+import nl.avisi.kotlinwebtest.ExecutionContext
+import nl.avisi.kotlinwebtest.Executor
+import nl.avisi.kotlinwebtest.StepRequest
+import nl.avisi.kotlinwebtest.StepResponse
+import nl.avisi.kotlinwebtest.TestCase
+import nl.avisi.kotlinwebtest.TestStep
+import nl.avisi.kotlinwebtest.expressions.ExpressionEvaluator
+import nl.avisi.kotlinwebtest.expressions.findExpressions
+import org.slf4j.LoggerFactory
+import java.io.IOException
+import java.sql.DriverManager
+import java.sql.ResultSet
+import java.sql.SQLException
+import java.util.ArrayList
+import java.util.HashMap
+
+
+class JdbcStepRequest(var body: String? = null) : StepRequest, DatabaseRequest() {
+
+    override infix fun url(url: String) {
+        testStep.url = url
+    }
+
+    override infix fun driver(driver: String) {
+        testStep.driver = driver
+    }
+
+    lateinit var testStep: JdbcTestStep
+}
+
+class JdbcStepResponse(override val resultSet: List<HashMap<String, String>>?,
+                       override val success: Boolean,
+                       override val message: String? = null) : DatabaseStepResponse
+
+class JdbcTestStep(testCase: TestCase) : TestStep<JdbcStepRequest, JdbcStepResponse>(testCase, JdbcStepRequest()) {
+    var url: String? = null
```

Comment:
volgens mij kan dit ook val zijn.

Reply:
Dit is niet mogelijk omdat er geen standaardwaarden zijn. er kan bijvoorbeeld een postgres en een mysql database request gemaakt worden.